<launch>
  <node pkg="obst_avoid" name="apf_mpc_test" type="apf_mpc_test" respawn="true" output="screen"/>
</launch>
